### PR TITLE
Automatically install Ubisoft Connect for multiple old Ubisoft games that still ship the uPlay installer

### PR DESCRIPTION
**Pull request by UsernamesAreNotMyThing**
Both games ship with an outdated and non-functional uPlay launcher, that needs to be replaced with Ubisoft Connect, which can be installed through Protontricks.

A lot of reports at https://www.protondb.com/app/33230?device=any and https://www.protondb.com/app/368500?device=any support this (especially the former), and it looks like this fix is still needed.

### CODE REVIEW

File: gamefixes-steam/368500.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""Game fix for Assassin's Creed: Syndicate"""
+
+from protonfixes import util
+
+
+def main() -> None:
+	"""Game ships with outdated and not working uPlay launcher."""
+	util.protontricks('ubisoftconnect')
```
Comment (R1kaB3rN) [inline gamefixes-steam/368500.py:8]: Thanks for this.

Also, following the upstream report for this title, I believe another environment seems to also be needed? Namely, PROTON_DISABLE_NVAPI=1

See https://github.com/ValveSoftware/Proton/issues/2464

Comment (UsernamesAreNotMyThing) [reply gamefixes-steam/368500.py]: Going by the second to last comment, it looks like there may already be plans to add that to official Proton.

**Comment by UsernamesAreNotMyThing** [comment]
Looks like Trackmania Turbo also needs this: https://github.com/ValveSoftware/Proton/issues/6637#issuecomment-1852468171

File: gamefixes-steam/368500.py (hunk 1 above)
Comment (R1kaB3rN) [reply gamefixes-steam/368500.py]: We can remove that specific fix once those plans ever come to fruition. For environment variables, there's generally no harm setting it twice. There's only harm when the env var isn't needed anymore and a proper upstream solution has been implemented to resolve the issue. In those cases, we need to pay attention.

File: gamefixes-steam/375900.py [hunk 2]
```diff
@@ -0,0 +1 @@
+33230.py
```
Comment (R1kaB3rN) [inline gamefixes-steam/375900.py:1]: Create a separate fix for this. I believe we only want to create symbolic links within the same dir if the titles are part of the same series

**Review by R1kaB3rN — CHANGES_REQUESTED** [review]

**Comment by R1kaB3rN** [comment]
For what it's worth, for users playing Trackmania Turbo, I'm referencing the Openplanet issue.

https://github.com/openplanet-nl/issues/issues/609

**Pull request merged**